resultsdbupdater/utils.py

```diff
@@ -9,6 +9,26 @@
 from . import config, exceptions
 
 
+MAX_RESULT_DATA_SIZE = 8192
```

Comment:
Could you set add the unit as a comment?

Should this value be configurable? And why 8192?

Reply:
I don't know how to find the actual limit -- I've reached the limit with `"." * 233963` set in a field but different one when trying to cut short the real message that caused the issue in prod.

Reply:
I saw this today:
OperationalError: (OperationalError) index row size 3032 exceeds maximum 2712 for index "result_data_idx_key_value"

Reply:
So maybe we should consider using this limit, even though it seems not that high....